Header header
float64 a
float64 b
float64 c
float64 d
float64 num_points
int64 color_id
geometry_msgs/Pose pose